tensorpack/dataflow/format.py

```diff
@@ -103,6 +118,8 @@ def get_data(self):
                 if k != '__keys__':
                     yield [k, v]
         else:
+            if not hasattr(self, "rng"):
+                self.reset_state()
```

Comment:
Really? I never did that before. I will look into that again. But this would be Monday.